release/src/main/scripts/set_version.sh

```diff
@@ -24,7 +24,7 @@
 
 set -e
 
-function usage() {
+function usage(){
```

Comment:
Does this cause a problem?

Reply:
No, removing. This is just where the "dash" interpreter dies. Thanks!